README.md

```diff
@@ -10,19 +10,32 @@ FAR uses a fence agent to fence a Kubernetes node. Generally, fencing is the pro
+FAR remediates by executing a fence agent action, `reboot` or `off`, on the unhealthy node, and moving any remaining workloads to other nodes, so they can continue running and be isolated from the unhealthy node.
+
+### Reboot-based remediation
+
+The reboot is done by executing a fence agent for the unhealthy node while evicting the workloads from this node is achieved by tainting the node and deleting the workloads. FAR unique taint, `medik8s.io/fence-agents-remediation`, has a [NoExecute effect](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions), so  any pods that don't tolerate this taint are evicted immediately, and they won't be scheduled again after the node has been rebooted as long as the taint remains (the taint is removed on FenceAgentsRemediation CR deletion). Deleting the workloads is done to speed up Kubernetes rescheduling of the remaining pods (most likely [stateful pods](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#using-statefulsets)), that are not running anymore.
+
+### Power-off-based remediation
+
+If you choose the `off` action, FAR powers off the unhealthy node using the fence agent. Unlike `reboot`, the node does not automatically return to service. It remains powered off until manually recovered by an administrator. The taint and workload eviction behavior remain the same as in the `reboot` case, ensuring that the node is isolated and workloads are safely rescheduled elsewhere.
 
 FAR includes the `FenceAgentsRemediationTemplate` (or `fartemplate`) Custom Resource Definition ([CRD](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)) for how to create a FenceAgentsRemediation CR. The template has the same fields as far CR (e.g., agent name) and it is used for automatically creating remediation CR by another operator/mechanism (e.g., [NHC](#far-with-nhc)). The other operator is responsible of creating (and eventually deleting) the FenceAgentsRemediation CR with the name of the unhealthy node, even though FAR can be used manually without fartemplate and an additional operator (see [standalone FAR](#standalone-far)).
```

Comment:
NIT: The following line could be interpreted under the new `### Power-off-based remediation` subsection.

It might be better to have a new subsection for  `fartemplate` so this line won't be considered as only associated with power-off remediation.